CONTRIBUTING.md

```diff
@@ -42,8 +42,8 @@ brew style foo
 ```
 
 After testing, if you think it is needed to force the corresponding bottles to be
-rebuilt and redistributed, add a line of the form `revision 1` to the formula,
-or add 1 to the revision number already present.
+rebuilt and redistributed, add a line of the form `revision 1` to the formula above the `head` line,
```

Comment:
Not every formula has a `head` line so this is a bit confusing 🤔. Maybe we could recommend `brew style --fix` instead which will fix this up?

Reply:
Perhaps
```suggestion
rebuilt and redistributed, add a line of the form `revision 1` to the formula,
or add 1 to the revision number already present. Run `brew style foo` again, and if desired include the `--fix` flag to automatically correct the formula to standard style.
```

Alternatively, is it important that the `revision` tag be suggested only after install/testing? Could it not be included in the earlier section around L31:

```
If you are already well versed in the use of `git`, then you can find the local
copy of the `homebrew-core` repository in this directory
(`$(brew --prefix)/Homebrew/Library/Taps/homebrew/homebrew-core/`).
Modify the formula there, leaving the section `bottle do ... end` unchanged, and if you think it is needed to force the corresponding bottles to be rebuilt and redistributed, add a line of the form `revision 1` to the formula,
 or add 1 to the revision number already present.
Prepare a pull request as you usually do.  Before submitting your pull request, be sure to test it
with these commands:
```

Reply:
Yeh, it feels like it makes more sense earlier. I like the idea of just recommending `brew style --fix` before committing 👍